mu-plugins/blocks/multisite-latest-posts/src/block.js

```diff
@@ -53,13 +55,27 @@ const Block = ( { endpoint, perPage } ) => {
 		return <div>{ error }</div>;
 	}
 
+	let domParser;
+	if ( 'function' === typeof DOMParser ) {
+		domParser = new DOMParser();
+	}
+
 	return (
 		<ul className="wporg-multisite-latest-posts">
 			{ posts.map( ( i ) => {
+				let title;
```

Comment:
This would remove the need for the `else`
```suggestion
				let title = i.title.rendered;
```

Reply:
👍🏻